osbrain/helper.py

```diff
@@ -193,3 +193,33 @@ def wait_agent_attr(agent, name='received', length=None, data=None, value=None,
             break
         time.sleep(0.01)
     return False
+
+
+def wait_agent_condition(agent, condition, *args, timeout=3, **kwargs):
+    """
+    Wait for an agent's condition to be true.
+
+    This condition is passed as a function whose only parameter is the actual
+    agent.
```

Comment:
`whose only parameter is ...` seems to be wrong (since we can pass more parameters around). What about something like: `The condition is passed as a function which must take, at least, the actual agent as a parameter, and must evaluate to a boolean result`.